example-raft-kv/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "example-raft-kv"
+version = "0.1.0"
+edition = "2021"
+authors = [ "drdr xp <drdr.xp@gmail.com>", ]
+categories = ["algorithms", "asynchronous", "data-structures"]
+description = "An example distributed kv-store built upon `openraft`."
+#documentation = "https://docs.rs/memstore"
+homepage = "https://github.com/datafuselabs/openraft"
+keywords = ["raft", "consensus"]
+license = "MIT/Apache-2.0"
+repository = "https://github.com/datafuselabs/openraft"
+readme = "README.md"
+
+[[bin]]
+name = "raftkv"
+path = "src/bin/raftkv.rs"
+
+
+[dependencies]
+actix-web = "4.0.0-rc.2"
+anyerror = { version = "0.1.1"}
+async-trait = "0.1.36"
+clap = { version = "3.0.13", features = ["derive", "env"] }
+env_logger = "0.9.0"
+openraft = { version="0.6", path= "../openraft" }
```

Comment:
I would set the crates source to prevent any conflict with future versions, we can keep the demo updated accordingly.

Reply:
I do not get it. What do you want it to look like? 🤔

Reply:
I mean replace that with `openraft = "0.6.4"` to force cargo to get the build from the stable release, but I  noticed that you did not that because there are changes to the core, understandable for the moment.